scss/_patterns_code-copyable.scss

```diff
@@ -20,13 +20,13 @@ $cc-button-size: 36px;
 
     &::before {
       color: $color-mid;
```

Comment:
It seems that currently many of these styles on `::before` were needed for sake of rendering $ as text (for example `color`, `line-height`, but possibly more). We should keep only things that are necessary to style the icon.

This will likely result in icon not being added as `content`, but as background-image, so the `::before` needs to be correctly sized and positioned as well.

Reply:
good point!